packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/actions.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+#[cfg_attr(feature = "web", derive(tsify::Tsify))]
+#[cfg_attr(feature = "web", tsify(from_wasm_abi))]
+pub struct ActionBody<T> {
```

Comment:
What function does `ActionBody` perform? Does it help with the types? A simpler alternative would to just put the action args directly in the action `enum` for each component. `ActionBody` does currently appear to do anything in the rust side.